GOVERNANCE.md

```diff
@@ -89,6 +89,19 @@ The aspiration is no one company or organization should have oversight of the ov
 
 Oversight Committee members are publicly listed in the `@gitops-working-group/oversight-committee` <https://github.com/orgs/gitops-working-group/teams/oversight-committee> GitHub team.
 
+### Working Group Co-Chairs
+
+The GitOps Working Group draws together individuals who are collaborating on forming content and technology "to provide companies and individuals with the skills, knowledge and competency to implement GitOps tooling and methodologies which simplify the operation and management of infrastructure and cloud native applications" and to care for the OpenGitOps  project.
+
+Co-chairs:
+
+- Ensure that meetings and other activities are conducted and progress continues to be made against the project agenda
+- Have GitOps domain knowledge
+- Will engage deeply in the work being done by the working group - that is, the role is not purely administrative. They are, however not responsible for performing all of the work - this is shared across the working group members and expected from those who have volunteered onto various workstreams.
```

Comment:
Wondering if we should use the term "technical coordination" somewhere to emphasize that this isn't just an administrative role ?